### ISSUE TITLE
Error logging to syslog

### ISSUE THREAD

**Issue by marco-moraschinelli** [pear/Log#33]
Trying to write a log to syslog is generated this error:

**Typed property Log_syslog::$name must be int, string used**

In syslog.php there is a different type of variable $name in class declaration:

```
class Log_syslog extends Log
{
    /**
     * Integer holding the log facility to use.
     */
    private int $name;
```

 and costructor:
```

    public function __construct(
        string $name,
```

**Comment by jparise** [comment]
@sveneld, could you take a look at this?

**Comment by jparise** [comment]
Thanks for the bug report!

This is now fixed in version [1.14.4](https://github.com/pear/Log/releases/tag/1.14.4).
